docs/guide/bootloader.md

```diff
@@ -12,10 +12,10 @@ nav_order: 1
   <img loading="lazy" src="/assets/images/bl-unlock.png" width="30%"/>
 </p>
 
-Unlocking your bootloader is a fundamental step when customizing your device, as it allows you to flash and install unofficial images in it. Please note this procedure **will completely wipe your data**, so make a backup before proceeding.
+Unlocking your bootloader is the first step when installing a custom rom your device, as it allows you to flash and install unofficial images on it. Please note this procedure **will completely wipe your data**, so make a backup before proceeding.
```

Comment:
Typo: "installing a custom rom your device". I also didn't specify the custom ROM use case as that's not the only reason to unlock the bootloader

Reply:
Yeah but imo, the older one which said "fundamental step when customising your phone" isnt exactly true

Reply:
In c9f0856c08de81eefdf0b731b75b20eb0eacf5bc, what about "the first step to follow to customize your device"?

Reply:
customising the device could also just be changing the launcher/modding using something like the goodlock suite, which is why im hesitant to use that word

Reply:
You're right, maybe "installing a custom ROM or simply rooting your device" fits better?

Reply:
yeah sounds good, made the change, +1 to go ahead and merge this from my side.